WordPressComKit/Me.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public class Me {
+    var ID: Int!
+    var displayName: String?
+    var username: String!
+    var email: String?
+    var primaryBlogID: Int?
+    var primaryBlogURL: NSURL?
+    var language = "en"
```

Comment:
Why? Or more generally, should we have default values at all on models? Since the model is being generated from API data, the fields should be there. If they're not, it's an error and a default value won't help us.